### PR TITLE
Updating DAST openAPI path

### PR DESCRIPTION
**Pull request by JohnNKing**
# Updating DAST openAPI path

Correcting the OpenAPI path used by ZAP to perform DAST scans.

## Checklist

n/a

### CODE REVIEW

File: app/src/main/java/gov/hhs/cdc/trustedintermediary/external/javalin/App.java [hunk 1]
```diff
@@ -31,6 +31,10 @@ public class App {
     public static void main(String[] args) {
         var app = Javalin.create().start(8080);
 
+        // apply this security header to all responses, but allow it to be overwritten by a specific
+        // endpoint by using `before` if needed
+        app.before(ctx -> ctx.header("X-Content-Type-Options", "nosniff"));
```
Comment (JohnNKing) [inline app/src/main/java/gov/hhs/cdc/trustedintermediary/external/javalin/App.java:36]: @halprin Ah! That's much cleaner.

**Review by jorg3lopez — APPROVED** [review]
Great work! I left a comment as well.

**Review by basiliskus — APPROVED** [review]

File: .github/workflows/dast.yml [hunk 2]
```diff
@@ -40,18 +40,12 @@ jobs:
       - name: Check API Health
         run: ./health-check.sh
 
-      - name: Pull ZAP OWASP Image
-        run: docker pull owasp/zap2docker-weekly
-
-      - name: Create docker network
-        run: docker network create zapnet
```
Comment (jorg3lopez) [inline .github/workflows/dast.yml:47]: Just wondering why the net was taken out.

Comment (halprin) [reply .github/workflows/dast.yml:47]: Ah, yeah.  If the network was going to be used, the TI service also needed to run inside the network.  It wasn't.  So the network wasn't buying us anything.  Instead the TI service is being exposed to the host network which is why we do the entire `ip -f inet -o addr show docker0 | awk '{print $4}' | cut -d '/' -f 1` thing to get the host IP address.

**Pull request merged**